config/src/aws-net-std__gnat.adb

```diff
@@ -114,51 +108,64 @@ package body AWS.Net.Std is
       Host          : String      := "";
       Reuse_Address : Boolean     := False;
       IPv6_Only     : Boolean     := False;
-      Family        : Family_Type := Family_Unspec)
-   is
-      use type Sockets.Family_Type;
-      Addr_Info : Sockets.Address_Info;
-      Created   : Boolean := False;
+      Family        : Family_Type := Family_Unspec) is
    begin
       if Socket.S /= null then
          Socket := Socket_Type'(Net.Socket_Type with others => <>);
       end if;
 
       Socket.S := new Socket_Hidden;
 
-      Addr_Info := Get_Inet_Addr (Host, Family, Passive => True);
+      declare
+         use GNAT.Sockets;
+         Error : Exceptions.Exception_Occurrence;
```

Comment:
This one also needs alignment.

Reply:
Done, renamed Error to Keep_Excp to be consistent with another similar block.